iocore/eventsystem/I_EventProcessor.h

```diff
@@ -99,12 +101,22 @@ class EThread;
 class EventProcessor : public Processor
 {
 public:
+  using EThreadInitFxn_t = Thread::EThreadInitFxn_t;
+  using ThreadFxn_t = std::function<void(const EThreadInitFxn_t &)>;
+
   /** Register an event type with @a name.
 
       This must be called to get an event type to pass to @c spawn_event_threads
       @see spawn_event_threads
    */
-  EventType register_event_type(char const *name);
+  EventType register_event_type(char const *name)
```

Comment:
Why make this inline? It's not called often, it's cleaner to have it in the .cc file.